datahub-web-react/src/app/entity/shared/Properties.tsx

```diff
@@ -1,16 +1,19 @@
 import { Space, Table, Typography } from 'antd';
 import React from 'react';
+import { ColumnsType } from 'antd/es/table';
 import { StringMapEntry } from '../../../types.generated';
 
 export type Props = {
     properties: StringMapEntry[];
 };
 
 export function Properties({ properties }: Props) {
-    const propertyTableColumns = [
+    const propertyTableColumns: ColumnsType<StringMapEntry> = [
         {
             title: 'Name',
             dataIndex: 'key',
+            sorter: (a, b) => b.key.localeCompare(a.key),
```

Comment:
should this be `a.key.localeCompare(b.key)`?

Reply:
Yes, indeed. Nice catch. The combined `b.key.localeCompare(a.key)` and `descend` resulted in ascending order. I've pushed a fix